python_toolbox/evaluation/README.md

```diff
@@ -13,11 +13,9 @@ https://tanksandtemples.org/download/
 
 **Step 2**. Install Open3D. Follow instructions in http://open3d.org/docs/getting_started.html
 
-**Step 3**. Clone this repository and follow instructions in ``setup.py``.
-
-**Step 4**. Run the evaluation script and grab some coffee.
+**Step 3**. Run the evaluation script and grab some coffee.
 ```
-python run.py
+python run.py --dataset-dir C:/TanksAndTemples/evaluation/data/Ignatius --traj-path C:/TanksAndTemples/evaluation/data/Ignatius/Ignatius_COLMAP_SfM.log --ply-path C:/TanksAndTemples/evaluation/data/Ignatius/Ignatius_COLMAP.ply
```

Comment:
"C:" is Windows style folder. How about removing "C:"?

Reply:
I think I adapted it from something already existent, but I changed it to something more generic.